config/locales/views/reactions/en.yml

```diff
@@ -17,7 +17,7 @@ en:
           other: "%{count}%{start}\_reactions%{end}"
         title: Reactions
       like:
-        title: Heart
+        title: Like
```

Comment:
another possible deletion?

Reply:
And these i18n keys are used on app/views/articles/_actions.html.erb, app/views/notifications/shared/_article_preview.html.erb and app/views/notifications/shared/_comment_box.html.erb

If you consider it, we can use a single yml file for this task

Reply:
I have taken a look at what has been proposed, and I think it is better to leave it as it is for the sake of comprehensibility.